RecoParticleFlow/PFProducer/data/download.url

```diff
@@ -1,8 +1,3 @@
-http://cmsdoc.cern.ch/cms/data/CMSSW/RecoParticleFlow/PFProducer/data/MVAnalysis_BDT.weights_finalID_hzz-pions.txt
```

Comment:
this file can be removed.
We are not using download.url technology since a while